scss/elements/_form.scss

```
@@ -153,105 +153,113 @@ $components-colors: (
 	height: 2.5rem;
 }
 
-// // input icon
-// .#{$prefix}input.-has-icon-left,
-// .#{$prefix}select.-has-icon-left,
-// .#{$prefix}input.-has-icon-right {
-// 	position: relative;
-// 	> i {
-// 		display: flex;
-// 		align-items: center;
-// 		justify-content: center;
-// 		position: absolute;
-// 		top: 0px;
-// 		width: 42px;
-// 		height: 100%;
-// 	}
-// }
+// input icon
+.#{$prefix}input.-has-icon-left,
+.#{$prefix}select.-has-icon-left,
+.#{$prefix}input.-has-icon-right {
+	position: relative;
+	> span.icon {
+		color: $color-border-input;
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		position: absolute;
+		top: 0px;
+		width: 42px;
+		height: 100%;
+		font-size: 1em;
+	}
+}
 
-// .#{$prefix}input.-has-icon-left {
-// 	> input {
-// 		padding-left: 42px;
-// 	}
-// 	> i {
-// 		left: 0px;
-// 	}
-// }
+.#{$prefix}input.-has-icon-left {
+	> input {
+		padding-left: 42px;
+	}
+	> span.icon.-is-left {
```

Add empty line before new rule

---

unnecesory specific selector 
```suggestion
	> .icon {
```